src/steps/emacs.rs

```diff
@@ -77,35 +77,35 @@ impl Emacs {
     }
 
     pub fn upgrade(&self, ctx: &ExecutionContext) -> Result<()> {
-        let emacs = require("emacs")?;
-        let init_file = require_option(self.directory.as_ref(), String::from("Emacs directory does not exist"))?
-            .join("init.el")
-            .require()?;
-
         if let Some(doom) = &self.doom {
-            return Emacs::update_doom(doom, ctx);
+            Emacs::update_doom(doom, ctx)
```

Comment:
What's wrong with having `return`?

Reply:
not sure why i did this.

Take a look now.